AirCasting/SearchAndFollow/CompleteScreen/Contollers/SearchAndFollowCompleteScreenController.swift

```diff
@@ -0,0 +1,65 @@
+// Created by Lunar on 04/05/2022.
+//
+
+import Foundation
+import Resolver
+
+protocol SearchAndFollowCompleteScreenController {
```

Comment:
well, the `Controller` realm is really reserved for high-level stuff, domain only. Here we have a mix:
* `download` suggests networking, not really a high level scenario (`retrieve` would fit tho)
* Usage of `StreamWithMeasurement` which is a service layer struct

Reply:
So should it be called service?